src/containers/CourseCard/components/CourseCardActions/index.jsx

```diff
@@ -5,24 +5,22 @@ import { ActionRow } from '@openedx/paragon';
 
 import { reduxHooks } from 'hooks';
 
-import UpgradeButton from './UpgradeButton';
 import SelectSessionButton from './SelectSessionButton';
 import BeginCourseButton from './BeginCourseButton';
 import ResumeButton from './ResumeButton';
 import ViewCourseButton from './ViewCourseButton';
+import CourseCardActionSlot from '../../../../plugin-slots/CourseCardActionSlot';
```

Comment:
Nit: this can actually be imported like so:

```suggestion
import CourseCardActionSlot from 'plugin-slots/CourseCardActionSlot';
```
I'll admit I don't know exactly how this works though. It doesn't work for all MFEs so it must have something to do with the way LD is set up 🤷